js/backup_restore.js

```diff
@@ -714,7 +752,11 @@ function configuration_restore(callback) {
             }
 
             function send_led_strip_config() {
-                MSP.sendLedStripConfig(reboot);
+                MSP.sendLedStripConfig(save_to_eeprom);
+            }
+            
+            function save_to_eeprom() {
+                MSP.send_message(MSP_codes.MSP_EEPROM_WRITE, false, false, reboot);
```

Comment:
eeprom is now saved twice in a row... line 704 should only be a call to `send_led_strip_config()`